packages/atomic/src/locales.json

```
@@ -6723,5 +6723,59 @@
     "zh": "引用",
     "zh-CN": "引用",
     "zh-TW": "引用"
+  },
+  "copy-generated-answer": {
+    "en": "Copy answer",
+    "fr": "Copier la réponse",
+    "cs": "Kopírovat odpověď",
+    "da": "Kopiér svar",
+    "de": "Antwort kopieren",
+    "el": "Αντιγραφή απάντησης",
+    "es": "Copiar respuesta",
+    "fi": "Kopioi vastaus",
+    "hu": "Válasz másolása",
+    "id": "Salin jawaban",
+    "it": "Copia risposta",
+    "ja": "回答をコピー",
+    "ko": "답변 복사",
+    "nl": "Antwoord kopiëren",
+    "no": "Kopier svar",
+    "pl": "Skopiuj odpowiedź",
+    "pt": "Copiar resposta",
+    "pt-BR": "Copiar resposta",
+    "ru": "Скопировать ответ",
+    "sv": "Kopiera svar",
+    "th": "คัดลอกคำตอบ",
+    "tr": "Yanıtı kopyala",
+    "zh": "复制答案",
+    "zh-CN": "复制答案",
+    "zh-TW": "複製答案"
+  },
+  "generated-answer-copied": {
+    "en": "Copied!",
+    "fr": "Copié !",
```

There should be no whitespace before the exclamation mark.

```suggestion
    "fr": "Copié!",
```